VersionControlUI/src/DiffManager.cpp

```diff
@@ -331,11 +330,16 @@ void DiffManager::createOverlaysForChanges(Visualization::ViewItem* diffViewItem
 		if (newNodeItem)
 			allItemsToScale.insert(newNodeItem);
 
-		if (change.changeType_== FilePersistence::ChangeType::Move)
+		if (change.changeType_== FilePersistence::ChangeType::Move && newNodeItem && oldNodeItem)
```

Comment:
If one of the items is null it does not make sense to add an arrow between them. The if structure was changed in a later commit a bit. An outer if checks if the items are not null and only then checks the other stuff.